fluesterfix/__init__.py

```diff
@@ -44,8 +53,17 @@
         'download done': '&#x2705; File retrieved',
         'error': 'Error',
         'only once': 'You can only do this once.',
+        'request': 'Request a secret',
+        'request desc': 'Give the Request-Link (validity 7 days) below to someone you want '
+                        'to retrieve a secret from. You can reload this page '
+                        'or click on the ID to see if the secret is already '
+                        'stored. As long as you see this site the secret '
+                        'is not there yet',
         'reveal!': 'Reveal the secret',
         'reveal?': 'Reveal this secret?',
+        'rid missing': 'Request ID (rid) missing.',
+        'rid secret stored': 'Secret stored',
+        'rid secret stored desc': 'The requested secret can now be revealed on the other side.',
         'secret': 'Secret',
         'share new secret': 'Share a new secret',
         'share new file': 'Share a new file',
```

Comment:
I am not sure, but I did not find the link check for the 7 days validity?! Where is it? Or did you forget to implement it?